### PR TITLE
doc: detail `fs snapshot mirror daemon status` mgr command

### PR DESCRIPTION
**Pull request by vshankar**
@leseb noticed that this interface did not have any documentation. Thanks!

Signed-off-by: Venky Shankar <vshankar@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/dev/cephfs-mirroring.rst [hunk 1]
```diff
@@ -210,8 +210,44 @@ Mirror Daemon Status
...
 
 Mirror daemons get asynchronously notified about changes in file system mirroring status
-and/or peer updates. CephFS mirror daemons provide admin socket commands for querying
-mirror status. To check available commands for mirror status use::
+and/or peer updates.
+
+CephFS mirroring module provides `mirror daemon status` interface to check mirror daemon
+status::
+
+  $ ceph fs snapshot mirror daemon status <fs>
```
Comment (leseb) [inline doc/dev/cephfs-mirroring.rst:218]: The doc is not consistent about the naming, sometimes the filesystem name is referred to as `<fs>` and sometimes `<fs_name>`, would be nice to fix this at some point.

**Review by leseb — APPROVED** [review]
Thanks for the follow-up!

File: doc/dev/cephfs-mirroring.rst (hunk 1 above)
Comment (vshankar) [reply doc/dev/cephfs-mirroring.rst:218]: oh yeh! I was thinking the same when writing this line

;P

**Comment by vshankar** [comment]
@leseb please check -- s/<fs>/<fs_name>

**Review by leseb — APPROVED** [review]
Looks good!

**Review by batrick — CHANGES_REQUESTED** [review]
Please fix the commit title to just be `doc:` or `doc/dev:`.

**Review by rishabh-d-dave — APPROVED** [review]

**Comment by vshankar** [comment]
> Please fix the commit title to just be `doc:` or `doc/dev:`.

done

**Review by batrick — APPROVED** [review]

**Comment by batrick** [comment]
jenkins test make check

**Review by batrick — CHANGES_REQUESTED** [review]
One last thing, please make a tracker ticket so this gets backported to Pacific. (Normally we don't do this for docs but we're moving in the direction that users no longer look at master documentation.)

**Review by batrick — APPROVED** [review]

**Pull request merged**